### DISCUSSION TITLE
Re: How to add Philips Hue Motion Sensors to Magic Occupancy

### DISCUSSION THREAD

**Discussion by planetmarsalarm** [Jason-Morcos/homebridge-magic-occupancy#9] [Q&A]
Having a difficult time figuring out how to add 2 Philips Hue Motion sensors to trigger the Occupancy Sensor and start the occupancy "Stay Occupied Delay", is the next step to add a Homekit automation that turns off the room lights when Magic Occupancy Sensor has a "Occupancy Detected No" state?

**Comment by planetmarsalarm** [comment]
Apologize issue was in Homebridge, deleted and reinstalled to resolve.  Plugin working well so far and was able to figure out the configuration.  THANK YOU!

**Reply by Jason-Morcos** [reply]
Glad you figured it out! No worries :)
